libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/http/DefaultHttpRequestComposer.kt

```diff
@@ -147,22 +149,33 @@ class DefaultHttpRequestComposer(
         operation: Operation<D>,
         customScalarAdapters: CustomScalarAdapters,
         sendApqExtensions: Boolean,
+        sendEnhancedClientAwarenessExtensions: Boolean,
         query: String?,
     ): Map<String, Upload> {
       return composePostParams(
-          writer, operation, customScalarAdapters, query, apqExtensionsWriter(operation.id(), sendApqExtensions)
+          writer, operation, customScalarAdapters, query, extensionsWriter(operation.id(), sendApqExtensions, sendEnhancedClientAwarenessExtensions)
       )
     }
 
-    private fun apqExtensionsWriter(id: String, sendApqExtensions: Boolean): JsonWriter.() -> Unit {
+    private fun extensionsWriter(apqId: String, sendApqExtensions: Boolean, sendEnhancedClientAwarenessExtensions: Boolean): JsonWriter.() -> Unit {
```

Comment:
Is there a preferred formatting style to stay within a column limit for these long function definitions?

Reply:
No preferrence from my side. We have the formatting rules in source control [here](https://github.com/apollographql/apollo-kotlin/tree/main/.idea/codeStyles). If you "reformat all" (code -> reformat file...) and it stays like this, I'm down with this.